octoprint/snapcraft.yaml

```diff
@@ -0,0 +1,21 @@
+name: octoprint
+version: 0.1
+architectures: [all]
+summary: The snappy web interface for your 3D printer
+description: The snappy web interface for your 3D printer
+
+apps:
+    octoprint:
+        command: octoprint
```

Comment:
This might need to be `usr/bin/octoprint` for it to work. Do you have `octocprint` in `/snap/bin`?